pipe.go

```diff
@@ -30,6 +31,12 @@ import (
 //sys rtlDosPathNameToNtPathName(name *uint16, ntName *unicodeString, filePart uintptr, reserved uintptr) (status ntStatus) = ntdll.RtlDosPathNameToNtPathName_U
 //sys rtlDefaultNpAcl(dacl *uintptr) (status ntStatus) = ntdll.RtlDefaultNpAcl
 
+type PipeConn interface {
```

Comment:
is this for convenience so you can write `c.(winio.PipeConn)` instead of `c.(interface{ Disconnect() error })`?